.github/workflows/studio_demo_test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         submodules: true
     
     - name: Build
-      run: scripts\build_studio_demo.cmd
+      run: scripts\build_guix_studio.cmd
```

Comment:
does it make sense for the name of build_guix_studio.cmd and test_studio_demo.cmd?

Reply:
This script is used for building GUIX Studio executable, it was first created for studio demo test, but now, the installer generator also use this script for building GUIX Studio executable. So rename it to build_guix_studio is more fitting.